components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/ext/listener/ExtensionListenerUtil.java

```diff
@@ -267,9 +267,8 @@ private static boolean handleExtensionResponse(MessageContext messageContext,
         //Extract customProperty from DTO
         Map<String, Object> customProperty = extensionResponseDTO.getCustomProperty();
         //set Additional analytics Properties to the message context.
-        if(customProperty != null && customProperty.get(APIMgtGatewayConstants.ADDITIONAL_ANALYTICS_PROPS) != null ) {
-            Map<String, Object> customElkAttributes =
-                    (Map<String, Object>) customProperty.get(APIMgtGatewayConstants.ADDITIONAL_ANALYTICS_PROPS);
+        if(customProperty != null && customProperty.get(APIMgtGatewayConstants.ADDITIONAL_ANALYTICS_PROPS) != null) {
```

Comment:
Need space after 'if'. @dimuthnc Please use code formatter plugin to format automatically.

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/11772/commits/970a6a2f8c745789ead1b8fd3bf4d0846cf21845